plugins/module_utils/common.py

```diff
@@ -265,28 +274,6 @@ def diff_objects(existing, new):
             result['after'] = diff[1]
         return not diff, result
 
-
-class KubernetesAnsibleModule(AnsibleModule, K8sAnsibleMixin):
```

Comment:
We can probably leave this and the `KubernetesRawModule` classes around as dummies with warnings to avoid a breaking API change